proto/plan.proto

```diff
@@ -218,6 +219,43 @@ message PropertiesDef {
 	repeated Property properties 	= 1;
 }
 
+
+enum PartitionType {
```

Comment:
if only used in PartitionInfo.   you can move these code to message PartitionInfo

Reply:
I've tried to do this, but the constant name generated by protobuffer is very lengthy and does not express the meaning